src/MeshKernel/CurvilinearGridOrthogonalization.cpp

```diff
@@ -52,13 +49,14 @@ meshkernel::CurvilinearGridOrthogonalization::CurvilinearGridOrthogonalization(s
     m_d.resize(m_grid->m_numM, std::vector<double>(m_grid->m_numN, doubleMissingValue));
     m_e.resize(m_grid->m_numM, std::vector<double>(m_grid->m_numN, doubleMissingValue));
     m_atp.resize(m_grid->m_numM, std::vector<double>(m_grid->m_numN, doubleMissingValue));
+    m_isGridNodeFrozen.resize(m_grid->m_numM, std::vector<bool>(m_grid->m_numN, false));
```

Comment:
here you are direcly accessing an attribute of another class (m_grid->m_numN) 
you could consider writig a getter function for it and use that instead: this reduces coupling between the classes

Reply:
Hi, I understand the reason behind it. The approach of direct access of class members has been used elsewhere to avoid writing getter and setters for all class properties (which in this sort of numerical library are quite a lot). A change in the approach should be consistent across all files.